cli/src/tasks/new-plugin.ts

```diff
@@ -234,13 +235,13 @@ function generatePackageJSON(answers: NewPluginAnswers) {
     author: answers.author,
     license: answers.license,
     dependencies: {
-      '@capacitor/core': 'latest'
+      '@capacitor/core': `${cliVersion}`
```

Comment:
We should allow a version range, otherwise it will install many different `@capacitor/core` packages.

```suggestion
      '@capacitor/core': `^${cliVersion}`
```